includes/plugins.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Utility functions to handle Platform plugins.
+ *
+ * @package hm-platform
+ */
+
+namespace HM\Platform\Plugins;
+
+use HM\Platform as Platform;
+use HM\Platform\Config as Config;
+
+/**
+ * Get a list of all available plugins.
+ *
+ * @return array Plugins and the associated configuration.
+ */
+function get_available_plugins() {
+	return Config\get_config()['plugins'];
+}
+
+/**
+ * Get a list of all enabled plugins.
+ *
+ * @return array Plugins and the associated configuration.
+ */
+function get_enabled_plugins() {
+	return array_filter( get_available_plugins(), function( $plugin ) {
+		return isset( $plugin['enabled'] ) && $plugin['enabled'] === 'yes';
+	} );
+}
+
+/**
+ * Load all enabled plugins, along with their customisation files.
+ */
+function load_enabled_plugins() {
+	foreach ( get_enabled_plugins() as $plugin => $data ) {
+		if ( ! empty( $data['prependFile' ] ) ) {
+			require WP_CONTENT_DIR . '/' . $data['prependFile'];
```

Comment:
I think this is alright as it gives us some other possibilities for drop in type behaviour if we added `file_exists() / is_readable()` checks in here.

On the other hand it is useful for someone to see that something is broken, perhaps an error log or throwing an exception would a stronger way to do that to prevent any broken code being committed.

Reply:
👍 for the checks.